src/System.Security.Cryptography.Xml/ref/Configurations.props

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp;
-      uap;
+      netstandard;
+      uap
```

Comment:
@weshaggard: Here are the exact errors when uap10.1 is in pkgproj and ref and src have only netstandard